BCStudentSoftwareDevTeam/celts
# Time check

Addresses issue #768 
Did:
- Check to make sure start time is not after end time
- Made sure start time is not the same as the end time
- Added a clearer error message
Test:
- Create an event- make it recurring, make end time a time before the start time
- Once you try to create event, the error message "Event end time must be after start time." should pop up 
- Create an event- make it recurring, make end time the same as start time
- Once you try to create event, the error message "Event end time must be after start time." should pop up
